src/components/asynccontent/AsyncContent.js

```diff
@@ -37,10 +39,7 @@ export default class AsyncContent extends React.Component {
 
   renderLoading() {
     return (
-      <div>
-        <h2>Loading</h2>
-        <FontAwesome name="spinner" size="5x" spin />
-      </div>
+      <LoadingSpinner />
```

Comment:
+1. Let's move `LoadingSpinner` into `components/asynccontent`. I don't think it needs it's own directory